src/main.leo

```diff
@@ -293,3 +293,29 @@ function mint(owner: address, amount: u64, totalsupply_amount: u64, gates:u64, b
 	};
 	return (newBalance, newtotalsupply);
 }
+
+@program
```

Comment:
It's good to have an explanation of the purpose of each test function and its parameters.

Reply:
that´s right. I´ll resolve it.